aiden/class1-1/10951.py

```diff
@@ -0,0 +1,8 @@
+#10951
+
+while True:
+    try:
+        a, b = map(int, input().split())
+        print(a + b)
+    except:
+        break
```

Comment:
```suggestion
        print(a + b)
    except EOFError:
        break
```

예측 가능한 에러를 핸들링 할 때는 대응하고자 하는 에러를 명시하는것이 보다 좋을것 같습니다.